production/helm/loki/values.yaml

```diff
@@ -56,6 +56,8 @@ config:
       directory: /data/loki/index
     filesystem:
       directory: /data/loki/chunks
+  querier:
+    max_look_back_period: '720h'
```

Comment:
why different default value?

Reply:
And no need to quote it.

Reply:
You mean this should be 0? Same as what is set in RegisterFlags [here](https://github.com/sandlis/loki/blob/query-max-lookback/pkg/querier/querier.go#L28)

Reply:
Yes, the defaults should be the same, one of the reasons we are adding it to helm is to make it configurable by the user, but if the user doesn't change anything, loki should start with the defaults.